.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
+
+(Tell us what the problem is in a few word)
+
+# Your environment
```

Comment:
This I would move to the bottom. 

It is in fact the least important part, so it should not take up the top

Reply:
Did it in https://github.com/inmanta/inmanta/pull/2416/commits/97df07c2c9d53836b23046f865629bacef6439c7